ui.tests/test-module/specs/wizard/wizard.authoring.spec.js

```diff
@@ -155,4 +163,51 @@ describe('Page - Authoring', function () {
         });
 
     });
+
+    context('Test Wizard Component String Language', function (){
+        beforeEach( () => {
+            cy.visit('http://localhost:4502/aem/start.html');
+            cy.get('#username').type('admin');
+            cy.get('#password').type('admin');
+            cy.get('#submit-button').click();
+            cy.on('uncaught:exception', () => {
+                return false
+            });
+
+            changeLanguage("de");
+            cy.visit('http://localhost:4502/libs/fd/fm/gui/content/forms/af/create.html').then(() => {
+                // Your tests here
+                cy.wait(30000);
```

Comment:
We generally don't use `cy.wait` instead, we wait for a particular event to be triggered. 
If cy.wait is the only option then 30 second delay is huge, it should be improved!

Reply:
Updated with cy.wait('@templates',{requestTimeout: 30000});